### DISCUSSION TITLE
%E4%B8%9A%E5%8A%A1%E6%B5%81%E7%A8%8B/%E6%B8%A0%E9%81%93%E7%AE%A1%E7%90%86/%E8%AE%BE%E7%BD%AE%E7%83%AD%E9%97%A8%E4%BA%A7%E5%93%81

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#586] [Announcements]
# %E4%B8%9A%E5%8A%A1%E6%B5%81%E7%A8%8B/%E6%B8%A0%E9%81%93%E7%AE%A1%E7%90%86/%E8%AE%BE%E7%BD%AE%E7%83%AD%E9%97%A8%E4%BA%A7%E5%93%81

一、业务背景 报件场景下因为金融产品较多，选取时用户体验交叉。 二、业务描述 2.1、使用群体 经销商金融专员。 2.1、场景&用途 设置经销商热门金融产品，在报件场景下可优先展示，提升用户体验。热门产品序号优先级数值越小，产品越热门。 2.2、业务对象 经销商金融产品。 2.3、业务流程
 TODO 三、功能介绍 ```mermaid flowcha...

http://localhost:8080/%E4%B8%9A%E5%8A%A1%E6%B5%81%E7%A8%8B/%E6%B8%A0%E9%81%93%E7%AE%A1%E7%90%86/%E8%AE%BE%E7%BD%AE%E7%83%AD%E9%97%A8%E4%BA%A7%E5%93%81.html

**Comment by RexusCao** [comment]
厉害！
